learn-accessibility-by-building-a-website-for-people-with-disabilities/css/style570.css

```diff
@@ -0,0 +1,142 @@
+/*
+  Let's make sure that all elements have adequate contrast to comply with WCAG
+  (Web Content Accessibility Guidelines).
+
+  WCAG defines two levels of compliance: AA (acceptable) and AAA (best).
+
+  As we're aiming for the best accessibility here, let's aim for AAA
+  compliance.
+
+  First, ensure that the foreground `color` of '.top-bar a' and the
```

Comment:
I noticed your `checkContrasts()` function in the JS file. However, I think it would be better prompt students to apply the colors directly. It's possible that they're working through the curriculum from beginning to end, and may not have gone through any of the JS courses yet. Also, I don't see a rule targeting `.top-bar a` for some reason.